python/cudf/dataframe/dataframe.py

```diff
@@ -399,6 +403,9 @@ def to_string(self, nrows=NOTSET, ncols=NOTSET):
         >>> df.to_string()
         '   key   val\\n0    0  10.0\\n1    1  11.0\\n2    2  12.0'
         """
+        if isinstance(self.index, cudf.dataframe.multiindex.MultiIndex) or\
+           isinstance(self.columns, cudf.dataframe.multiindex.MultiIndex):
+            raise TypeError("You're trying to print a DataFrame that contains a  MultiIndex. Print this dataframe with .to_pandas()")  # noqa: E501
```

Comment:
Suggest fixing the formatting so there isn't a needed for a `noqa` as well.
```suggestion
            raise NotImplementedError("We don't yet support printing DataFrames that contain a MultiIndex. You can use `.to_pandas()` to print this dataframe using Pandas.")
```